UE/Application/Ports/UserPort.cpp

```diff
@@ -30,6 +30,10 @@ void UserPort::showNotConnected(){
     gui.showNotConnected();
 }
 
+void UserPort::showPeerUeBecomesUnknown(common::PhoneNumber phoneNumber){
+    gui.showPeerUserNotAvailable(phoneNumber);
```

Comment:
Tutaj jedna zmiana, daj jakiegoś sleepa na 2 sekudny, bo to się w ogóle nie wyświetla.